src/transformers/utils/quantization_config.py

```diff
@@ -33,6 +34,11 @@
 logger = logging.get_logger(__name__)
 
 
+class QuantizationMethod(Enum):
```

Comment:
You might need to also subclass str to be able to serialize the config after.

Reply:
nice catch